mock-issuer-service/.env.example

```
@@ -0,0 +1,22 @@
+# Copy to .env and adjust. All values are optional; defaults shown below.
+
+# --- Issuer server ---
+PORT=4000
+HOST=0.0.0.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HOST key should go before the PORT key

(UnorderedKey)